### ISSUE TITLE
Treeview -  JAWS doesn't narrate position for each tree item

### ISSUE THREAD

**Issue by kolaps33** [FreedomScientific/VFO-standards-support#338]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html
> 2. Navigate in the tree through the item on first and second level

### Expected result
I would expect that JAWS narrate the position on each tree item for particular level. 

### Actual result
Item position is narrated only on first focused item. 

Narration on first level:
Fruits closed 1 of 3
To move through or expand items use the Arrow keys.
Vegetables closed
Grains closed

Narration on next level:
Oranges 1 of 5
To move through or expand items use the Arrow keys.
Pineapple  
Apples closed
Bananas 
Pears closed

### Example
https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html

### Additional Information
Chrome + NVDA narrate position for each tree item on particular level

#### JAWS version and build number
2019.1909.28

 
#### Operating System and version
Windows 10 Enterprise
version 1909

#### Browser and version:
Chrome  78.0.3904.70

**Comment by JAWS-test** [comment]
The problem also occurs with IE 11 and Firefox. JAWS only outputs the current position when the hierarchy level changes. This doesn't have to be the first entry, it can also be the last entry or an entry in the middle (e.g. "3 of 6"). I'm not sure if this should be considered an error or if JAWS outputs the level only if it's important.

**Comment by JeremiahRogersVispero** [comment]
This is designed behavior intended to balance verbosity and efficiency.
